marlowe-runtime-web/src/Language/Marlowe/Runtime/Web/Contract/State.hs

```diff
@@ -0,0 +1,44 @@
+module Language.Marlowe.Runtime.Web.Contract.State
+  ( ContractState(..)
+  , Payout(..)
+  ) where
+
+
+import Data.Aeson (FromJSON, ToJSON)
+import Data.Map (Map)
+import Data.OpenApi (ToSchema)
+import Data.Text (Text)
+import Data.Word (Word64)
+import GHC.Generics (Generic)
+import qualified Language.Marlowe.Core.V1.Semantics.Types as Semantics
+import Language.Marlowe.Runtime.Web.Orphans ()
+
+import Language.Marlowe.Runtime.Web.Types
+  (BlockHeader, MarloweVersion, Metadata, PolicyId, TextEnvelope, TxOutRef, TxStatus)
+
+data ContractState = ContractState
+  { contractId :: TxOutRef
+  , roleTokenMintingPolicyId :: PolicyId
+  , version :: MarloweVersion
+  , continuations :: Maybe Text
+  , tags :: Map Text Metadata
+  , metadata :: Map Word64 Metadata
+  , status :: TxStatus
+  , block :: Maybe BlockHeader
+  , initialContract :: Semantics.Contract
+  , currentContract :: Maybe Semantics.Contract
+  , state :: Maybe Semantics.State
+  , utxo :: Maybe TxOutRef
+  , txBody :: Maybe TextEnvelope
+  , unclaimedPayouts :: [Payout]
```

Comment:
Here we could reuse the existing `PayoutRef` type, which also includes the `assets`

Reply:
Oh snap, no it doesn't!

Reply:
Everything is missing the `assets`... I've gotta fix that!